src/components/pages/LoginPage/interface.ts

```diff
@@ -16,6 +16,7 @@ export interface User {
   id: number;
   email: string;
   token: string;
+  expiry?: number;
```

Comment:
Yeah, here as well the user interface makes me think : ) Which things are needed inside the context :)